loom/engine/loom2d/l2dRectangle.h

```diff
@@ -260,15 +260,18 @@ class Rectangle
     /**
      * Make a copy of this Rectangle.
      */
-    Rectangle *clone()
+    int clone(lua_State *L)
```

Comment:
Will do. I didn't to a lot of searching, but a lot of the code uses the same code, mostly vector stuff :)